sockeye/constants.py

```diff
@@ -238,7 +238,8 @@
 # Arguments that may differ and still resume training
 ARGS_MAY_DIFFER = ["overwrite_output", "use-tensorboard", "quiet",
                    "align_plot_prefix", "sure_align_threshold",
-                   "keep_last_params"]
+                   "keep_last_params", "seed", "max-updates",
+                   "max-num-epochs", "max-samples"]
```

Comment:
How about `--min-num-*`? In general, would it make sense to make a pass over current arguments to see whether we should add more to this list? Its a suboptimal solution as it becomes outdated quickly, but if you are at it anyway right now :)

Reply:
Agree, ```--min-*``` shouldn't matter either, added them.